Formula/dub.rb

```diff
@@ -2,10 +2,10 @@ class Dub < Formula
   desc "Build tool for D projects"
   homepage "https://code.dlang.org/about"
   url "https://github.com/dlang/dub.git",
-      :tag => "v1.0.0",
-      :revision => "b59af2b8befb4fad4157d8c9cc86dba707b2fc87"
+      tag: "v1.0.0",
```

Comment:
If I do that then `brew audit --new-formula dub` fails, saying "Use the new Ruby 1.9 hash syntax."